tests/lightning_base/test_device.py

```diff
@@ -323,6 +325,41 @@ def test_dynamic_wires_from_circuit_fixed_wires(self, circuit_in, n_wires, wires
             assert device._statevector._num_wires == n_wires
             assert device._statevector._wires == qml.wires.Wires(range(n_wires))
 
+    @pytest.mark.parametrize(
+        "circuit_0, n_wires_0",
+        [
+            (QuantumScript([qml.RX(0.1, 0)], [qml.expval(qml.Z(1))]), 2),
+            (QuantumScript([qml.RX(0.1, 0), qml.RX(0.1, 1)], [qml.expval(qml.Z(2))]), 3),
+        ],
+    )
+    @pytest.mark.parametrize(
+        "circuit_1, n_wires_1",
+        [
+            (QuantumScript([qml.RX(0.1, 0)], [qml.expval(qml.Z(1))]), 2),
+            (QuantumScript([qml.RX(0.1, 0), qml.RX(0.1, 1)], [qml.expval(qml.Z(2))]), 3),
```

Comment:
Could you also test this with different number of qubits and shots?
```suggestion
            (QuantumScript([qml.RX(0.1, 0)], [qml.expval(qml.Z(1))]), 3),
            (QuantumScript([qml.RX(0.1, 0), qml.RX(0.1, 1)], [qml.expval(qml.Z(2))]), 5),
```

Reply:
updated